modules/seed/templates/bootkube/bootstrap-manifests/bootstrap-apiserver.yaml

```diff
@@ -36,6 +36,13 @@ spec:
     - --tls-cert-file=/etc/kubernetes/secrets/apiserver.crt
     - --tls-private-key-file=/etc/kubernetes/secrets/apiserver.key
     - --kubelet-preferred-address-types=InternalIP,ExternalIP,Hostname
+    - --requestheader-allowed-names=aggregator
```

Comment:
In `modules/seed/templates/bootkube/manifests/kube-apiserver.yaml` you are using `- --requestheader-allowed-names=system:kube-aggregator` while you are using `- --requestheader-allowed-names=aggregator` here.